core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -71,6 +71,11 @@ trait HighPriorityToResponseInstances extends LowPriorityToResponseInstances {
   ): Aux[AsyncStream[A], Text.Plain] =
     asyncResponseBuilder((a, cs) => e(a, cs).concat(newLine))
 
+  implicit def sseAsyncToResponse[A](implicit
```

Comment:
I think we can define this under `ServerSentEvent` companion object and it should be resolved just fine. Any chance you can try this?